packages/edit-navigation/docs/user-documentation.md

```diff
@@ -0,0 +1,72 @@
+Note: this documentation is a work in progress.
+
+# Block-based menu editor
+
+The Block-based Navigation Editor brings the power of blocks to the Appearance > Navigation section in the WordPress Administration Screens allowing you use blocks to create menus.
```

Comment:
Are we going to use the term `menu` or `navigation`? Menu doesn't sort of bind us to the classic menus screen...

Reply:
Yeah, I don't know! This question was also asked in the tracking issue. The button is currently 'New menu' in the editor, so I think as longs as the docs match the UI it's ok.

If/when the UI changes we can update the docs to match.

I personally can't see a switch entirely to 'Navigation' because 'New navigation' doesn't sound right. That might be because navigation is an uncountable noun, so it can't represent a tangible item.

Reply:
Oh whoops, I see now you were referring to the title. Good point, I changed it.